lib/broadside/ecs/ecs_deploy.rb

```diff
@@ -171,22 +171,23 @@ def update_task_revision
     # reloads the service using the latest task definition
     def update_service
       task_definition_arn = EcsManager.get_latest_task_definition_arn(family)
-      debug "Updating #{family} with scale=#{@target.scale} using task #{task_definition_arn}..."
+      debug "Updating #{family} with scale=#{@target.scale} using task_definition #{task_definition_arn}..."
 
       update_service_response = EcsManager.ecs.update_service({
         cluster: @target.cluster,
-        desired_count: @target.scale,
+        desired_count: @target.scale.to_i,
```

Comment:
we do
```
validates :scale, numericality: { only_integer: true }
```
unfortunately this is ActiveModel, so it considers `"1"` to be a valid integer as well as the actual `1`, so i added the `to_i`.  we're at least guaranteed that it's either an actual integer or a string with a valid integer in it.